include/net/lwm2m.h

```diff
@@ -1145,5 +1145,19 @@ void lwm2m_rd_client_update(void);
  */
 char *lwm2m_path_log_strdup(char *buf, struct lwm2m_obj_path *path);
 
+/* 
+ * @brief LwM2M SEND operation to given path list
+ *
+ * @param ctx LwM2M context
+ * @param content_format 110 for SenML Json format or 112 for SenML CBOR
+ * @param path_list Path list for triggered Resources
+ * @param path_list_size Length of path list. Max size is defined by CONFIG_LWM2M_COMPOSITE_READ_MAX_URL
+ *
+ * 
+ * @return 0 for success or negative in case of error.
+ *
+ */
+int lwm2m_engine_send(struct lwm2m_ctx *ctx, uint16_t content_format, char *path_list[], uint8_t path_list_size);
```

Comment:
Why do we need content format as a parameter here?
I would assume that it is job for engine to choose which one to use from built in formats. You most probably will build in only one content format.

Reply:
Can the `path_list` be `const char *`.
Also, can I free the list after this call? It should be documented how long I need to keep that in memory.

Reply:
Content format can be internally selected by engine I'm fine with that change. 

Path list need to at memory only time at function call. String path is converted to to internal used object path structure.